components/script/dom/element.rs

```diff
@@ -1190,23 +1190,35 @@ impl Element {
 
     // Element branch of https://dom.spec.whatwg.org/#locate-a-namespace
     pub fn locate_namespace(&self, prefix: Option<DOMString>) -> Namespace {
-        let prefix = prefix.map(String::from).map(LocalName::from);
+        let ns_prefix = prefix.clone().map(|s| Prefix::from(&*s));
```

Comment:
I would go with `namespace_prefix` as `ns` is a bit ambigous.

```suggestion
        let namespace_prefix = prefix.clone().map(|s| Prefix::from(&*s));
```